lib/scsi_defs.h

```diff
@@ -80,5 +83,37 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 73 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  // All fields big-endian
+  struct ModeSense6Command {
+    OpCode opCode = OpCode::kModeSense6;
+    uint8_t reserved_1 : 4;
+    bool dbd : 1;
```

Comment:
nit: having a short comment for non-obvious field names would be helpful in the future. Non-obvious field names include abbreviated names like `dbd`, `pc` etc